lib/adapters/index.js

```diff
@@ -3,8 +3,11 @@ var Juttle = require('../runtime/index').Juttle;
 var stochastic = require('./stochastic-adapter');
 var file = require('./file-adapter');
 var http = require('./http');
+var io = require('./io');
 
 // register the built-in adapters
 Juttle.adapters.register('file', file({}));
 Juttle.adapters.register('http', http({}));
 Juttle.adapters.register('stochastic', stochastic({}));
+Juttle.adapters.register('stdin', io.StdinAdapter({}));
```

Comment:
humm I guess its not a bad thing since I can't right now see a reason to want to write to stderr and therefore `read io` reads from stdin and `write io` writes to stdout and if someone really wants the option to write to stderr then we'd simply add the option `write io -stderr true` for that very special case.